js/testing_1_2_3.js

```diff
@@ -0,0 +1,14 @@
+// https://www.codewars.com/kata/testing-1-2-3/train/javascript
+
+var number=function(array){
+  let i = 1;
```

Comment:
There is no need to create a variable for the iterator :smile: Take a look at what parameters the function passed to `array.map` receives and try to work with that instead of having an `i` variable